File: Iniciante/1116/1116.c
#include <stdio.h>
 
int main() {
 
    int x, y, N, i;
    
    scanf("%d", &N);
 
    for(i=0;i<N;i++) {
        scanf("%d%d", &x, &y);
        if (y == 0)
            printf("divisao impossivel\n");
        else
            printf("%0.1lf\n", (double)x/y);
    }
    
    return 0;
}

File: Python/Iniciante/1116.py
# Problema 1116 - Beecrowd - Iniciante - Nível 1
# Recebendo número de divisões na entrada de dados
N = int(input())

# For que repete N vezes
for i in range(N):
    # Aplicação da função float através do map na lista de dois valores (X, Y)
    X, Y = map(float, input().split(' '))
    # Caso Y não seja igual a 0, é possível efetuar a divisão
    if Y != 0:
        print(X/Y)
    # Caso contrário, mostrar na tela a mensagem "divisao impossivel"
    else:
        print('divisao impossivel')
